### ISSUE TITLE
Future migration improvements

### ISSUE THREAD

**Issue by Jamie-SA** [semanticarts/gist#559]
Keeping track of these nice-to-have thoughts:

- [ ] [Possible Option] Create script/query to report on any terms that are in the gistDeprecated file
- [ ] [Possible Option] Create script/query to report on any terms in the gist namespace that are not in the current version of gist

**Comment by uscholdm** [comment]
@Jamie-SA @rjyounes 
Do we want to move forward on this?

**Comment by rjyounes** [comment]
Yes, I think we should leave this open. It's up to developer bandwidth how soon they get implemented.
